cmd/lambda/lambda.go

```diff
@@ -41,6 +41,10 @@ func HandleRequest(ctx context.Context) {
               Name: aws.String("GERRIT_PROJECT"),
               Value: aws.String("GERRIT_PROJECT"),
             },
+            &ecs.KeyValuePair{
+              Name: aws.String("GERRIT_PROJECT_FILTER"),
+              Value: aws.String("GERRIT_PROJECT_FILTER"),
+            },
```

Comment:
This need not be set.

Reply:
Explained in the previous comment - I prefer to be more explicit, so we know what is needed, you can just put `0` or `false` which does the same as not sending at all.